db/sqlglue.c

```diff
@@ -2233,8 +2217,8 @@ static void genid_hash_add(BtCursor *cur, int rrn, unsigned long long genid)
 
     /* printf("genid_hash_add(cur %d rrn %d genid %08llx rc %d)\n",
      * cur->cursorid, rrn, genid, rc); */
-    rc = bdb_temp_hash_insert(cur->bt->genid_hash, &key, sizeof(struct key),
-                              &genid, sizeof(genid));
```

Comment:
Does bdb_temp_hash_insert() log about failures? If not, may be we should check the return code and log an appropriate message?